02LessonTwo/homework/index.html

```diff
@@ -25,7 +25,7 @@
   </div>
     <hr>
     <div id="job">
-    <center><img src="https://cdn0.iconfinder.com/data/icons/ikooni-outline-seo-web/128/seo2-18-512.png" height="10%" width="10%"></center>
```

Comment:
The `center` element has been deprecated. In other words, its not valid HTML 5.
Use css, `img { display: block; margin: auto;}`
You can still use `center` but future project that you work on will require you to adhere to the HTML 5 specification. You can read this stackoverflow page, http://bit.ly/1veYbBb
Good job otherwise.